spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -762,6 +762,15 @@ class PodsProjectGenerator
               resource.should.be.nil
             end
 
+            it 'doesn\'t add xcassets resources to the static framework target' do
+              @pod_target.stubs(:build_type => BuildType.static_framework)
+              @installer.install!
+              resources = @project.targets.first.resources_build_phase.files
+              resources.count.should > 0
+              resource = resources.find { |res| res.file_ref.path.include?('Images.xcassets') }
+              resource.should.be.nil
+            end
+
```

Comment:
Could you add one more? For the dynamic framework, ensuring it does add the asset catalog?

Reply:
good idea, done! 😄